.gitignore

```diff
@@ -11,6 +11,7 @@ local.properties
 
```

Comment:
functional test: 
- I set Vehicle.Speed using the normal "VSS_PATH"-Mode (resp. using the cli: feed Vehicle.Speed 100), switched to the SPECIFICATION mode and selected Speed (as I couldn't spot a different one). When fetching this value I would expect it to return 100, however it does return a "VssSpeed(value=0.0)
- observing Vehicle.Speed and updating it using the CLI leads to the same result here doesn't matter the value, it always returns "VssSpeed(value=0.0)"

Reply:
Should be fixed now.

Reply:
Added pro guard rules for this case.